app/routes/organen/orgaan/index.js

```diff
@@ -1,24 +1,33 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
+import { getFormFrom } from 'frontend-lmb/utils/get-form';
+import { BESTUURSORGAAN_FORM_ID } from 'frontend-lmb/utils/well-known-ids';
+import RSVP from 'rsvp';
 
-export default class OrganenOrgaanMandatenRoute extends Route {
+export default class OrganenOrgaanIndexRoute extends Route {
   @service store;
 
   async model() {
-    const { currentBestuursorgaan } = this.modelFor('organen.orgaan');
+    const parentModel = this.modelFor('organen.orgaan');
 
-    const mandaten = await currentBestuursorgaan.bevat;
+    const mandaten = await parentModel.currentBestuursorgaan.bevat;
     const [orderedMandaten, availableBestuursfuncties] = await Promise.all([
       this.orderMandaten(mandaten),
       this.computeBestuursfuncties(mandaten),
     ]);
 
-    return {
+    const bestuursorgaanFormDefinition = getFormFrom(
+      this.store,
+      BESTUURSORGAAN_FORM_ID
+    );
+
+    return RSVP.hash({
```

Comment:
Do we need the RSVP here? All in information is awaited above? Or im still missing something about the RSVP ;p

Reply:
In general if you return multiple async resources it is best to use the RSVP hash, is also something that the ember guides suggest. In this case I added this, because the formDefinition was still undefined when I tried to access it in the template (I see there is no await there, could be that just adding the await before the getFormFrom also solves it, but don't necessarily think that is better, since you don't need this for fetching of other resources)